controllers/ccruntime_controller.go

```diff
@@ -166,23 +169,19 @@ func (r *CcRuntimeReconciler) setCleanupNodeLabels() (ctrl.Result, error) {
 	}
 
 	for _, node := range nodesList.Items {
-		for k, v := range node.GetLabels() {
-			if k == installDoneLabel[0] && v == "true" {
-				nodeLabels := node.GetLabels()
-				nodeLabels[k] = "cleanup"
-				node.SetLabels(nodeLabels)
-				break
+		labels := node.GetLabels()
+		if val, exists := labels[installDoneLabel[0]]; exists && val == "true" {
+			labels[StartUninstallLabel[0]] = StartUninstallLabel[1]
```

Comment:
Just double-checking that you removed `node.SetLabels()` call because `labels` is a pointer and the `nodesClient.Update()` call will update the node object according?

Reply:
This is not really my area but based on the experiments the `nodesClient.Update` sets the labels. Or should I add an explicit `node.SetLabels`?

Reply:
I don't know. It seems that `node.SetLabels` is redundant here, but I wanted to check with you it was dropped purposely.